Dockerfile

```diff
@@ -45,7 +50,7 @@ RUN git clone https://github.com/pirate/ArchiveBox /home/pptruser/app \
     && chown -R pptruser:pptruser /data \
     && ln -s /data /home/pptruser/app/archivebox/output \
     && ln -s /home/pptruser/app/bin/* /bin/ \
-    && ln -s /home/pptruser/app/bin/archive /bin/archivebox \
```

Comment:
Changed this line to the correct link (I had it backwards) instead of removing it:
`&& ln -s /home/pptruser/app/bin/archivebox /bin/archive \`